### ISSUE TITLE
@ember/string deprecation warnings

### ISSUE THREAD

**Issue by Windvis** [ember-cli/ember-resolver#853]
I noticed that the ember-canary scenario started showing a lot of deprecation warnings which seemed to be triggered be this addon.

```
DEPRECATION: Importing from `@ember/string` without having the `@ember/string` package in your project is deprecated. Please add `@ember/string` to your `package.json [deprecation id: ember-string.add-package] This will be removed in ember-source 5.0.0. See https://deprecations.emberjs.com/v4.x/#toc_ember-string-add-package for more details.
```

https://github.com/ember-cli/ember-resolver/blob/262a463bd20b8bbea48054a1c2a9689c1bc8ae32/addon/addon/resolvers/classic/index.js#L8

**Comment by kategengler** [comment]
@Windvis Thanks for opening the issue! Handling this (and any other @ember/string deprecations in a new app) is my next step.

**Comment by kategengler** [comment]
fixed by #862 and released in v10.0.0

**Comment by pzubar** [comment]
Thanks for this fix @kategengler!
Still, I have a question regarding the case when the only package that imports from `@ember/string` is `ember-resolver`. Should I still add `@ember/string` to my package.json to avoid getting the warning, or there is a better solution? Thank you in advance!
